pyvista/utilities/algorithms.py

```diff
@@ -0,0 +1,315 @@
+"""Internal vtkAlgorithm support helpers."""
+import traceback
+
+import numpy as np
+
+import pyvista
+from pyvista import _vtk
+from pyvista.errors import PyVistaPipelineError
+
+from .helpers import wrap
+
+
+def algorithm_to_mesh_handler(mesh_or_algo, port=0):
+    """Handle vtkAlgorithms where mesh objects are expected.
+
+    This is a convienance method to handle vtkAlgorithms when passed to methods
+    that expect a :class:`pyvista.DataSet`. This method will check if the passed
+    object is a ``vtk.vtkAlgorithm`` or ``vtk.vtkAlgorithmOutput`` and if so,
+    return that algorithm's output dataset (mesh) as the mesh to be used by the
+    calling function.
+
+    Parameters
+    ----------
+    mesh_or_algo : pyvista.DataSet or vtk.vtkAlgorithm or vtk.vtkAlgorithmOutput
+        The input to be used user input as a data set (mesh) or vtkAlgorithm object.
```

Comment:
Did you mean this, or something else?
```suggestion
        The input to be used as a data set (mesh) or vtkAlgorithm object.
```

Reply:
Ah yes, thank you for fixing... not sure what I was thinking when I wrote that 😵‍💫